openam-documentation/openam-doc-source/README.txt

```diff
@@ -21,7 +21,7 @@ $ mvn -Pbinary-licensing -Dcheckstyle.skip=true -DskipTests=true -DisDraftMode=n
 [INFO] ------------------------------------------------------------------------
 
 $ cd openam-documentation/openam-doc-source
-$ mvn -Dcheckstyle.skip=true -DskipTests=true -DisDraftMode=no -DreleaseVersion="11.0.0" -D"gaId=UA-23412190-14" -D"releaseDate=Software release date: November 08, 2013" org.forgerock.commons:forgerock-doc-maven-plugin:release
+$ mvn -Dcheckstyle.skip=true -DskipTests=true -DisDraftMode=no -DreleaseVersion="11.0.0" -D"gaId=UA-23412190-14" -D"releaseDate=Software release date: November 08, 2013" org.forgerock.ce.commons:forgerock-doc-maven-plugin:release
```

Comment:
I'm not sure we have released a community version of the forgerock-doc-maven-plugin, AFAIK it is public already: https://github.com/markcraig/forgerock-doc-maven-plugin -> although this one says out of date..
https://stash.forgerock.org/projects/DOCS/repos/forgerock-doc-maven-plugin/browse is not public, maybe @BohoCode knows what we should do about this.